File: AB-ArhivaBlanketa/app/src/main/java/com/ab/ssd/ab_arhivablanketa/Klase/Korisnik.java
package com.ab.ssd.ab_arhivablanketa.Klase;


import java.io.Serializable;

public class Korisnik implements Serializable {
    public int ID;
    public String ime;
    public String prezime;
    public int brIndexa;
    public String username;
    public String email;
    public String pass;
    public Fakultet fakultet;
    public boolean admin;

    public Korisnik(){}
    public Korisnik(int id, String ime, String prezime, int brIndexa, String username, String email, String pass, Fakultet faks, boolean admin)
    {
        this.ID=id;
        this.ime=ime;
        this.prezime=prezime;
        this.brIndexa=brIndexa;
        this.username=username;
        this.email=email;
        this.pass=pass;
        this.fakultet=faks;
        this.admin=admin;
    }
}

File: WEB-API/WEB-API/Models/Korisnik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WEB_API.Models
{
    public class Korisnik
    {
        public virtual int IdKorisnik { get; set; }
        public virtual string Ime { get; set; }
        public virtual string Prezime { get; set; }
        public virtual int BrIndeksa { get; set; }
        public virtual string Username { get; set; }
        public virtual string Password { get; set; }
        public virtual string Email { get; set; }
        public virtual bool Admin { get; set; }
        public virtual Fakultet Fakultet { get; set; }

        

    }
}